src/components/molecules/UiPhoneNumber/_internal/UiPhoneNumberInput/UiPhoneNumberInput.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <UiInput
+    v-model="phoneNumber"
+    :class="[
+      'ui-phone-number-input',
+      { 'ui-input--has-error': error },
+    ]"
+    :placeholder="placeholder"
+  >
+    <template #input>
+      <input
+        v-keyboard-focus
+        v-bind="inputAttrs"
+        :value="phoneNumber"
+        class="ui-input__input"
+        @keydown="keyValidation"
+        @input="inputHandler($event)"
+      />
+    </template>
```

Comment:
```suggestion
```
and that https://github.com/infermedica/component-library/pull/323/files#r1456037382

Reply:
I need this `<template #input>...` because I need to pass the same `id` for `label` and `input`. If I pass `attrs` to `UiInput` id is added to first element, not `input`